flocker/common/test/test_interface.py

```diff
@@ -0,0 +1,158 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+Tests for ``flocker.common.interface_decorator``.
+"""
+
+from twisted.trial.unittest import SynchronousTestCase
+
+from eliot.testing import (
+    LoggedMessage, assertContainsFields, capture_logging
+)
+from eliot import Field, MessageType
+
+from zope.interface import Interface, implementer
+
+from ..interface import interface_decorator
```

Comment:
After `interface` becomes `_interface` this should be changed to use the canonical, public name instead of importing from the private module.